dvc/remote/gdrive.py

```diff
@@ -195,10 +207,28 @@ def drive(self):
 
             # Pass non existent settings path to force DEFAULT_SETTINGS loading
             gauth = GoogleAuth(settings_file="")
-            gauth.CommandLineAuth()
 
-            if os.getenv(RemoteGDrive.GDRIVE_USER_CREDENTIALS_DATA):
-                os.remove(self.gdrive_user_credentials_path)
+            try:
+                gauth.CommandLineAuth()
+            except RefreshError as e:
+                raise GDriveAccessTokenRefreshError(
+                    msg="Google Drive's access token refreshment is failed",
+                    cause=e,
+                )
+            except KeyError as e:
+                raise GDriveMissedCredentialKeyError(
+                    msg="Google Drive's user credentials file '{}' "
+                    "misses value for key '{}'".format(
+                        self.gdrive_user_credentials_path, str(e)
+                    )
+                )
+            except (AuthenticationError, Exception) as e:
+                raise DvcException(
+                    "Google Drive authentication failed", cause=e
```

Comment:
can we use the same style everywhere = `(msg="msg", cause=e)`, or just `("msg", e)`. I prefer the second option tbh - since it's clear in this case what is being passed. Please, make all exception handlers the same.

Reply:
Have changed to ("msg", e) call format style, thanks!